packages/fxa-auth-client/lib/client.ts

```diff
@@ -168,19 +173,28 @@ export class AuthClientError extends Error {
 }
 
 export default class AuthClient {
+  // TBD: Should we change the version?
```

Comment:
For adding fields we haven't in the past.  I don't think it's worth doing now.

Except for VPN, we're the only consumer of this API.  Speaking of, how will the VPN be affected by this?  /cc @bbangert